Style fixes

melix99 opened the pull request:
See individual commits.

melix99 committed:
style: Make unread-count centered again

The bottom part was a bit taller after updating to recent GTK versions.

melix99 committed:
chat-action-bar: Simplify widget structure and fix style

Remove the GtkFrame, it was pretty useless. Also, the style was breaking the emoji popover of the textview, so I updated the css to be more precise with the selection of the right nodes.

melix99 committed:
user-dialog: Make the headerbar an overlay

This way it doesn't occupy all the size at the top.

marhkb commented on data/resources/style.css:
```diff
@@ -42,7 +42,7 @@
   min-width: 0.7em;
   border-radius: 10px;
-  padding: 2px 5px;
+  padding: 2px 5px 1px;
```
Here, at least, this is still not centered.
I have to set this to 0px instead

0px (left) vs 1px (right)
![Bildschirmfoto von 2022-01-06 23-08-59](https://user-images.githubusercontent.com/3630213/148459523-f4ffeca6-36d6-4e44-a54a-e096a428f8f4.png)

melix99 replied:
Right! Thanks for noticing that.

melix99 committed:
login: Improve qrcode style

Use the new .card class and simplify the widget tree. Also, don't hide the qrcode when pressing the previous button, it doesn't look that great.

melix99 commented on data/resources/ui/login.ui:
```diff
@@ -218,25 +218,12 @@
                                 <property name="halign">center</property>
-                                <property name="valign">center</property>
-                                <property name="child">
-                                  <object class="AdwBin" id="qr_code_bin">
```
Mh, thinking about that, maybe the intent was to not hide the entire card, but only hiding the image?

melix99 replied:
Actually, the fact that the qrcode hides when pressing the 'previous' button is not really great to see, I'll change that.

marhkb replied:
Agreed, hiding it was not the best idea.
Could we then at least apply a simple brightness filter? That makes it look more consistent when all other elements are insensitive.

For example, applying this
```css
.insensitive-image {
  filter: brightness(30%);
}
```
has quite a good result without too much effort.
![Bildschirmfoto von 2022-01-07 07-51-17](https://user-images.githubusercontent.com/3630213/148504160-853c8b1d-3c88-4e43-a2f7-3ff0f31fb8c2.png)

melix99 replied:
I just copied what libadwaita does (and also the difference in the hc style).
